### PR TITLE
[INCR-97] Upgrade dependencies in edx-submissions

### PR DESCRIPTION
**Pull request by cclauss**
Fixes: https://openedx.atlassian.net/browse/INCR-97

$ __python-modernize -w .__

Upgrading __ddt__ and __freezegun__ will fail the Travis tests.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @cclauss! I've created [OSPR-3120](https://openedx.atlassian.net/browse/OSPR-3120) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: pycodestyle [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[pycodestyle]
+exclude = ./.*,./docs/source/conf.py
```
[the remark this answers is not in the document]
Comment (cclauss) [reply pycodestyle]: Go ahead and time the runs with and without.  On a lightly used repo, timings will be similar but on repos with a large git history, they can be quite different.  Past experience has been that __.git__ and __.venv__ have both been problematic.  Also, note that these are patterns, not paths.

**Comment by natabene** [comment]
@cclauss Thank you for your contribution. @edx/incr-reviewers Can you give this a look?

**Comment by jmbowman** [comment]
I'm going to go ahead and merge this one as is; it does most of the work needed, I'll create separate tickets for the ddt and freezegun upgrades since it looks like each of those will require some corresponding code changes.

**Comment by edx-webhook** [comment]
@cclauss 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-submissions&entry.867055334=https://github.com/edx/edx-submissions/pull/91&entry.1484655318=https://github.com/cclauss&entry.752974735=2019-02-22+22:25&entry.1917517419=2019-03-22+17:26&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**